docs/source/notebooks/mmm/mmm_budget_allocation_example.ipynb

```diff
@@ -1729,7 +1729,8 @@
    "name": "python3"
   },
   "language_info": {
-   "name": "python"
+   "name": "python",
+   "version": "3.10.14"
```

Comment:
We can remove this notebook from the PR as there is no significant change

Reply:
Done!